### ISSUE TITLE
#if 1 or #if 0 is plain text instead of comment

### ISSUE THREAD

**Issue by Supernerd11** [atom/language-c#115]
what i want is
```cpp
#if 1
int hello()
#else
int goodbye()
#endif

#if 0
int hello()
#else
int goodbye()
#endif
```
where it comments out the unused part, but it just puts  both `#if 1` and `#if 0` as plain text.

**Comment by MaximSokolov** [comment]
This was fixed by #91. Atom stable v1.4.3 should contain fix

**Comment by 50Wliu** [comment]
Thanks @MaximSokolov.
